src/routes/github/configuration/github-configuration-post.ts

```diff
@@ -32,6 +36,30 @@ export const GithubConfigurationPost = async (req: Request, res: Response): Prom
 			});
 		return;
 	}
+	if (await booleanFlag(BooleanFlags.ENABLE_GITHUB_SECURITY_IN_JIRA, installation.jiraHost)) {
+		await submitSecurityWorkspaceToLink(installation.jiraHost, gitHubInstallationId, gitHubAppId, req.log);
+	}
 
 	res.sendStatus(200);
 };
+
+const submitSecurityWorkspaceToLink = async (
+	jiraHost: string,
+	gitHubInstallationId: number,
+	gitHubServerAppIdPk: number | undefined,
+	logger: Logger
+) => {
+
+	try {
+		const gitHubAppClient = await createAppClient(logger, jiraHost, gitHubServerAppIdPk, { trigger: "github-configuration-post" });
+
+		logger.info("Fetching info about installation");
+		const { data: installation } = await gitHubAppClient.getInstallation(gitHubInstallationId);
+
+		const jiraClient = await getJiraClient(jiraHost, gitHubInstallationId, gitHubServerAppIdPk, logger);
+		await jiraClient.linkedWorkspace.submit(installation.account.id);
+	} catch (err) {
+		logger.warn("Failed to submit security workspace to Jira", err);
```

Comment:
Nit: to align with other logging `({ err }, "Failed to submit security workspace to Jira");